packages/strapi-plugin-i18n/admin/src/permissions.js

```diff
@@ -0,0 +1,8 @@
+const i18nPermissions = {
```

Comment:
❓ I'm not sure about the shape of this object. It looks to resolve correctly the permissions however 🤷🏻‍♂️